lms/views/lti_launches.py

```diff
@@ -90,7 +90,7 @@ def launch_lti(request, lti_key=None, context_id=None, document_url=None,
     # Never prevent a launch because of logging problem.
     # pylint: disable=broad-except
     except Exception as e:
-        log.error(f"Failed to log lti launch for lti key '{lti_key}': {e}")
+        log.error("Failed to log lti launch for lti key '%s': %s", lti_key, e)
```

Comment:
`logging` supports passing in a string with `%s`;s as the first argument, then the values to be substituted as the second argument, and does the actual substituting itself, lazily. You're supposed to do this instead of formatting the string yourself (with an f-string or otherwise) before passing it in.